pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -1631,6 +1632,25 @@ var _ = Describe("Converter", func() {
 			Entry("and add the graphics and video device if it is set to true on arm64", True(), 1, "arm64"),
 			Entry("and not add the graphics and video device if it is set to false on arm64", False(), 0, "arm64"),
 		)
+
+		It("Should have one vnc", func() {
+			vmi := v1.VirtualMachineInstance{
+				ObjectMeta: k8smeta.ObjectMeta{
+					Name:      "testvmi",
+					Namespace: "default",
+					UID:       "1234",
+				},
+				Spec: v1.VirtualMachineInstanceSpec{
+					Domain: v1.DomainSpec{},
+				},
+			}
+
+			domain := vmiToDomain(&vmi, &ConverterContext{AllowEmulation: true})
+			Expect(domain.Spec.Devices.Graphics).To(HaveLen(1))
+			Expect(domain.Spec.Devices.Graphics).To(ContainElement(gstruct.MatchFields(gstruct.IgnoreExtras, gstruct.Fields{
```

Comment:
Nice use of the matcher, but maybe this suffices too?

`Expect(domain.Spec.Devices.Graphics[0]["Type"].To(Equal("vnc"))`

Reply:
Maybe but as it is written now is more verbose if it fails...